dev-libs/paho-mqtt-c/paho-mqtt-c-1.3.12-r1.ebuild

```
@@ -59,9 +66,18 @@ src_test() {
 		return
 	fi
 
-	${EPYTHON} "${BUILD_DIR}"/test/"${MY_TEST_UTILS}"/interoperability/startbroker.py -c \
-			   "${BUILD_DIR}"/test/"${MY_TEST_UTILS}"/interoperability/localhost_testing.conf &
+	echo "RUNNING TESTS"
+	echo "Start test broker"
+	mv "${WORKDIR}"/"${MY_TEST_UTILS}"-"${MY_TEST_COMMIT}" "${WORKDIR}"/"${MY_TEST_UTILS}" || die
+	cd "${WORKDIR}"/"${MY_TEST_UTILS}/interoperability" || die
+	${EPYTHON} startbroker.py -c localhost_testing.conf &
```

We should be extra nice an cleanup all started child processes.

```suggestion
	${EPYTHON} startbroker.py -c localhost_testing.conf &
	local -r startbroker_pid=$!
```

And then, at the end of `src_test`:
```
kill ${startbroker_pid} ${mqttsas_pid}|| die